extensions/amp-dailymotion/1.0/test/test-component.js

```diff
@@ -0,0 +1,79 @@
+import * as Preact from '#preact';
+import {Dailymotion} from '../component';
+import {mount} from 'enzyme';
+import {expect} from 'chai';
+
+describes.sandboxed('Dailymotion preact component v1.0', {}, () => {
+  it('Renders', () => {
+    const wrapper = mount(
+      <Dailymotion
+        style={{width: 300, height: 200}}
+        videoId="x3rdtfy"
+      ></Dailymotion>
+    );
+
+    const iframe = wrapper.find('iframe');
+
+    expect(iframe.prop('src')).to.equal(
+      'https://www.dailymotion.com/embed/video/x3rdtfy?api=1&html=1&app=amp'
+    );
+
+    // Style propagated to container, but not iframe.
+    expect(wrapper.prop('style').width).to.equal(300);
+    expect(wrapper.prop('style').height).to.equal(200);
+  });
+
+  it('Pass correct param attributes to the iframe src', () => {
+    const wrapper = mount(
+      <Dailymotion
+        style={{width: 300, height: 200}}
+        videoId="x3rdtfy"
+        endscreenEnable={false}
+        sharingEnable={false}
+        uiHighlight="444444"
+        info={false}
+      ></Dailymotion>
+    );
+
+    const iframe = wrapper.find('iframe');
+
+    // Enscreen Enable attr
+    expect(iframe.prop('src')).to.contain('endscreen-enable=false');
+    // Sharing Enable attr
+    expect(iframe.prop('src')).to.contain('sharing-enable=false');
+    // UI Highlight attr
+    expect(iframe.prop('src')).to.contain('ui-highlight=444444');
+    // info attr
+    expect(iframe.prop('src')).to.contain('info=false');
```

Comment:
nit: you can make a variable if you find yourself referencing the same prop over and over
```suggestion
    const src = iframe.prop('src');
    expect(src).to.contain('endscreen-enable=false');
    expect(src).to.contain('sharing-enable=false');
    expect(src).to.contain('ui-highlight=444444');
    expect(src).to.contain('info=false');
```